src/__tests__/graph.test.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import { Graph } from '../graph'
+
+// Mocked components
+const MockNodeComponent = () => <div>Node Component</div>
+const MockLinkComponent = () => <div>Link Component</div>
+
```

Comment:
Both of these test cases can be moved to `src/__tests__/index.test.tsx`, we can keep them all together

Reply:
Done